libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -186,9 +186,9 @@ def get_alt_function(self, mcu, pin, function):
                     return 0
             return None
 
-        if function and function.endswith("_RTS") and (
+        if function and (function.endswith("_RTS") or function.endswith("_RTS_GPIO") or function.endswith("_CTS_GPIO")) and (
                 function.startswith('USART') or function.startswith('UART')):
-            # we do software RTS
+            # we do software RTS/CTS
```

Comment:
I don't see any code changes to implement software CTS. We'd also need a much higher RTS threshold on the flight controller side (its 16 bytes at the moment, and a software CTS will be very laggy)

Reply:
i've fixed it up